modules/formulize/include/functions.php

```diff
@@ -5375,18 +5513,18 @@ function formulize_db_escape($value) {
 }
 
 // THANKS TO baptiste.place@utopiaweb.fr on php.net for this conversion function:
-/** 
```

Comment:
Another odd removal/addition block where the code looks the same?

Reply:
I believe here it's just spacing issues.